### PR TITLE
MNT: Re-rendered with conda-smithy 2.3.0

### PR DESCRIPTION
**Pull request by damianavila**
Supersede #10 on my own fork.

### CODE REVIEW

**Comment by damianavila** [comment]
This is additionally releasing a previous unreleased version: 1.3.0

**Comment by damianavila** [comment]
@jakirkham do you have any idea what is heppening with the circle CI run on this one? It seems it is entering in a loop with py36 and start building again when in fact the package was already built.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,28 +19,19 @@ build:
 
 requirements:
   build:
-    - anaconda-client
-    - notebook >=4.2.0
     - python
     - setuptools
+    - notebook >=4.2.0
+    - anaconda-client
```
Comment (jakirkham) [inline recipe/meta.yaml:25]: Are these needed at build time? If not, would recommend dropping them.

**Comment by jakirkham** [comment]
Hmm...not sure. Without having looked too deeply, my gut feeling is there is either a dependency issue (i.e. missing Python 3.6 for some package) or some post-link script is failing.

File: recipe/meta.yaml (hunk 1 above)
Comment (damianavila) [reply recipe/meta.yaml:25]: `anaconda-client` is not need for building, removed.

Comment (damianavila) [reply recipe/meta.yaml:25]: Well, in fact it is build dependency...

Comment (jakirkham) [reply recipe/meta.yaml:25]: Interesting. Sorry to lead you off track then.

**Comment by jakirkham** [comment]
Also could be a post-link script of a dependent package.

**Comment by damianavila** [comment]
I think the issue it around around `anaconda-client`. I see in the py27 and py35 run the 1.5.1 version from conda-forge... but then in the py36 I see the 1.6.2 version coming from defaults...
Checking here: https://anaconda.org/conda-forge/anaconda-client/files I don't see a py36 package so that could be the issue, somehow...

**Comment by jakirkham** [comment]
We tried to build and deploy for Python 3.6 using 1.5.1, but deployment failed across the board. 😞 ( https://github.com/conda-forge/anaconda-client-feedstock/pull/4 ) Restarted the CIs, but I think the issue goes deeper. Tried to upgrade to a newer version, but that has been stymied by Windows issues. ( https://github.com/Anaconda-Platform/anaconda-client/issues/398 ) ( https://github.com/Anaconda-Platform/anaconda-client/issues/399 ) Not really sure how to proceed and lack the time near term to dig in further. Could really use some help from `anaconda-client` devs.

**Comment by damianavila** [comment]
Now it it failing on the other pythons... so it seems tied to the 1.6.2 version... interesting.

>Could really use some help from anaconda-client devs.

Will ping the devs internally here, trying to find out who is dealing with this codebase now :wink:

**Comment by jakirkham** [comment]
Awesome! Thanks for doing that. 😄

**Comment by damianavila** [comment]
Disabling the testing step until things get fixed upstream around `anaconda-client`. Will merge after everything (except the linter is green).

**Comment by damianavila** [comment]
It is interesting that it is now failing in the combination `anaconda-client` 1.6.1 and python3.6 on win...

**Comment by damianavila** [comment]
`anaconda-client 1.6.3` did the trick. The two failures above in the CIs are cancelled jobs because I pushed first to origin instead of my own branch. So merging this one now.

**Pull request merged**